assets/apps/calendar/init.js

```diff
@@ -60,6 +60,66 @@ navigator.serviceWorker.getRegistration('./').then(swReg => {
     window.parent.postMessage("sw-registration-failure", parentHost)
 })
 
+let supported = ["en-GB"];
+let enGB = {
```

Comment:
Could we include this as a separate json file that is fetched so it's easier to add new languages?

Reply:
sure, separate js file is also fine.